metadata.json

```diff
@@ -26,11 +26,11 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">=3.7.0 <4.0.0"
+      "version_requirement": ">=3.7.0 <4.3.0"
```

Comment:
4.3.0 was just released, is there a particular reason to stay below that version when testing?